contracts/multisig-prover/src/execute.rs

```diff
@@ -121,31 +123,50 @@ fn get_verifiers_info(deps: &DepsMut, config: &Config) -> Result<VerifiersInfo,
             msg: to_json_binary(&active_verifiers_query)?,
         }))?;
 
-    let participants = verifiers
-        .clone()
+    let min_num_verifiers = deps
+        .querier
+        .query::<Service>(&QueryRequest::Wasm(WasmQuery::Smart {
+            contract_addr: config.service_registry.to_string(),
+            msg: to_json_binary(&service_registry::msg::QueryMsg::GetService {
+                service_name: config.service_name.clone(),
+            })?,
+        }))?
```

Comment:
do we not have a client for the service_registry contract yet?

Reply:
sad